abi/aapcs64/aapcs64.rst

```diff
@@ -1469,13 +1481,12 @@ is mangled as
 
 .. code:: c
 
-   _Z1fu10__Int8x8_t
+    _Z1fu10__Int8x8_t
 
 A platform ABI may instead choose to treat the types as normal structure types, without a ``u`` prefix. For example, a platform ABI may choose to mangle the function above as:
 
 .. code:: c
-
-   _Z1f10__Int8x8_t
+    _Z1f10__Int8x8_t
```

Comment:
It seems you are missing a blank line. It might not be required for getting the right formatting, but please put it back for consistency with the style used in the rest of the sources.

Reply:
done